qcodes/instrument_drivers/QuTech/D5a.py

```diff
@@ -98,6 +103,11 @@ def __init__(self, name, spi_rack, module, inter_delay=0.1, dac_step=10e-3,
                                vals=Enum(*self._span_set_map.keys()),
                                docstring='Change the output span of the DAC. This command also updates the validator.')
 
+    def _set_dac_unit(self, unit: str) -> None:
```

Comment:
and for consistency `self._mV` should also be changed.

Reply:
I removed `self._mV` as it is not used.